package.json

```diff
@@ -37,7 +37,9 @@
     "dependencies": {
         "@actions/core": "~1.10.0",
         "@actions/exec": "~1.1.1",
+        "@actions/http-client": "~2.2.0",
         "@actions/tool-cache": "~2.0.1",
+        "jwt-decode": "^4.0.0",
```

Comment:
This package cause issues with NodeJS <= 18.
I also didn't find any usage of it. Can we remove it?
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'jwt-decode@4.0.0',
npm WARN EBADENGINE   required: { node: '>=18' },
npm WARN EBADENGINE   current: { node: 'v16.20.2', npm: '8.19.4' }
npm WARN EBADENGINE }
```

Reply:
Fixed!
btw, how did you see this warning? what did you run in order to get it?